pkg/pgmodel/model/label_list.go

```diff
@@ -7,37 +7,23 @@ package model
 import (
 	"fmt"
 
-	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v5/pgtype"
 	"github.com/timescale/promscale/pkg/pgmodel/model/pgutf8str"
 )
 
 type LabelList struct {
-	names  *pgutf8str.TextArray
-	values *pgutf8str.TextArray
+	names  pgtype.FlatArray[pgutf8str.Text]
```

Comment:
Label names can't have NULL character so it's better to just use `pgtype.Text`. I assume this was not originally part of this change but since we are touching those code lines I think it's good to use opportunity to improve/fix things.

Reply:
Actually it seems it was `pgtype.Text` before.